### DISCUSSION TITLE
Anyway to set auto remove peer/config with our set time limit ?

### DISCUSSION THREAD

**Discussion by RetakerMZ** [complexorganizations/wireguard-manager#352] [Q&A]
Hey i wondeer is there anyway to set auto remove peer/config on our own ? i want to auto remove 30 days or 7 days

**Comment by malikshi** [comment]
did you have tried the script?

**Reply by digim-s15** [reply]
maybe Retaker was asking about INDIVIDUAL timeout for each client. that will be useful
